neptunecontrib/api/chart.py

```diff
@@ -123,10 +125,24 @@ def log_chart(name, chart, experiment=None):
 
         try:
             from plotly import tools
-            chart = tools.mpl_to_plotly(chart)
+
+            # When Plotly cannot accurately convert a matplotlib plot, it emits a warning.
+            # Then we want to fallback on logging the plot as an image.
+            #
+            # E.g. when trying to convert a Seaborn confusion matrix or a hist2d, it emits a UserWarning with message
+            # "Dang! That path collection is out of this world. I totally don't know what to do with it yet!
+            # Plotly can only import path collections linked to 'data' coordinates"
+            with warnings.catch_warnings():
+                warnings.filterwarnings("error", category=UserWarning,
```

Comment:
Turn a matching warning into a thrown UserWarning exception which is then handled.

Using warnings is probably the only way to find out if Plotly can successfully convert a matplotlib plot - or am I missing something?